### PR TITLE
Added Access Permanently Closed doors to dread

### PR DESCRIPTION
**Pull request by steven11sjf**
- added the closed doors from latest ODR release
- added a new cosmetic option to toggle default/alternate texture
- updated changelog and fixed a minor typo
- fixed tests

### CODE REVIEW

File: randovania/gui/ui_files/dread_cosmetic_patches_dialog.ui [hunk 1]
```diff
@@ -278,6 +278,13 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="alt_closed">
+            <property name="text">
+             <string>Alternate Access Permanently Closed shield texture</string>
```
Comment (Miepee) [inline randovania/gui/ui_files/dread_cosmetic_patches_dialog.ui:284]: I'd also use quotation marks here tbh

File: randovania/games/dread/gui/dialog/dread_cosmetic_patches_dialog.py [hunk 2]
```diff
@@ -67,6 +67,7 @@ def connect_signals(self) -> None:
         self._persist_shield_type_update("alt_bomb", self.alt_bomb)
         self._persist_shield_type_update("alt_cross_bomb", self.alt_cross_bomb)
         self._persist_shield_type_update("alt_power_bomb", self.alt_power_bomb)
+        self._persist_shield_type_update("alt_closed", self.alt_closed)
```
[the remark this answers is not in the document]
Comment (steven11sjf) [reply randovania/games/dread/gui/dialog/dread_cosmetic_patches_dialog.py:70]: uhh, what's that?

[the remark this answers is not in the document]
Comment (Miepee) [reply randovania/games/dread/gui/dialog/dread_cosmetic_patches_dialog.py:70]: if you go to this branch, open the cosmetic window, then close rdv, checkout main and open the cosmetic window there, do you get an exception about it being unable to load the cosmetics? If yes, you need to change `randovania/interface_common/persisted_options.py`

**Comment by steven11sjf** [comment]
fixed requested changes

**Review by Miepee — APPROVED** [review]

**Pull request merged**